include/bluetooth/testing.h

```diff
@@ -60,6 +65,70 @@ void bt_test_cb_unregister(struct bt_test_cb *cb);
 void bt_test_mesh_net_recv(u8_t ttl, u8_t ctl, u16_t src, u16_t dst,
 			   const void *payload, size_t payload_len);
 
+/** Indicate the model has been bound
+ *
+ *  Called upon successful model bind.
+ *  This will call registered mesh_model_bound callbacks.
+ *
+ *  @param addr Remote address
+ *  @param model Model
+ *  @param key_idx AppKey index
+ */
+void bt_test_mesh_model_bound(u16_t addr, struct bt_mesh_model *model,
+			      u16_t key_idx);
```

Comment:
Isn't this an internal API? Only public APIs, i.e. those that apps call should reside under include/bluetooth/. It seems to me you're only calling this API from within the mesh stack.

Reply:
OK, I've move this to internal API